remo/profiles/tasks.py

```diff
@@ -83,6 +83,7 @@ def reset_rotm_nominees():
         nominees = UserProfile.objects.filter(is_rotm_nominee=True)
         for nominee in nominees:
             nominee.is_rotm_nominee = False
+            nominee.rotm_nominated_by = None
```

Comment:
Although this here is correct, I am starting to think that if we want to keep additional data, we should not reset the nominations every month but instead keep them for historical reasons. If this the case then we probably need to re-design the db schema for the ROTM. I am adding this comment as a TODO item for me to investigate this further.